### DISCUSSION TITLE
How to add custom tool for turtle?

### DISCUSSION THREAD

**Discussion by Exzept1on** [cc-tweaked/CC-Tweaked#1676] [Q&A] [answered]
How to add custom tool for turtle? For example: custom pickaxe, via kubeJS or custom mod, can u give me example or something like that pls

**Comment by Wojbie** [comment, marked as answer]
As a example you can look at [netherite tools datapack](https://github.com/cc-tweaked/netherite-tools/tree/main), which adds enchantable turtle upgrades for all netherite tools. It should display example how to make datapack that adds your specific tools to CC:T and how to enable custom "allow enchanted" and "consume durability" modes if you want that.

Kinda related: #1600

**Reply by Exzept1on** [reply]
solve my problem, thanks
